usr/local/bin/bastille

```diff
@@ -45,6 +45,7 @@ bastille_conf_check
 . /usr/local/etc/bastille/bastille.conf
 # Set default values for config properties added during the current major version:
 : "${bastille_network_pf_ext_if:=ext_if}"
+: "${bastille_network_pf_ext_if:=ext_if6}"
```

Comment:
I assume this variable is meant to be `${bastille_network_pf_ext_if6:=ext_if6}`?

Reply:
Yes, you are correct... I fixed it locally but forgot to push.

Reply:
@waseigo can you push the change so we can review it again?